src/index.ts

```diff
@@ -53,7 +53,12 @@ export {
 };
 
 export { ERC725Config, KeyValuePair, ProviderTypes } from './types';
-export { flattenEncodedData, encodeData } from './lib/utils';
+export {
+  flattenEncodedData,
+  encodeData,
+  encodePermissions,
+  decodePermission,
+} from './lib/utils';
```

Comment:
I suggest to add it in the class - so instead of export from - you can import from

And then add a static function within the class.